docs/usage-tui.md

```diff
@@ -198,8 +201,20 @@ So, to construct an output, do this:
 Note that receiver's address does not take quotes.
 
 Finally, send a transaction by giving `send` command a list of outputs (`out` argument), wallet
-passphrase (`pass` argument) if any and, optionally, wallet's name or index (`wallet` argument). If
-you don't specify a wallet, the selected one will be used. Here's an example:
+passphrase (`pass` argument) if any and, optionally, wallet's name or
+index (`wallet` argument) as well as list of source accounts (as names
+or indices). If you don't specify a wallet, the selected one will be used.
+
+Transaction inputs are selected automatically from a set of accounts
+which depends on `account` arguments and current selection:
+* If at least one account is explicitly specified, only the specified
+  accounts can be used as inputs.
+* If no accounts are explicitly specified and an account from the
+  input wallet is selected, only its addresses will be used as inputs.
+* Otherwise, addresses will be picked from all accounts in the input
```

Comment:
This seems to contradict help, right?

> If an account in the input wallet is selected, only this account will be used as input.

Reply:
> * If no accounts are explicitly specified and an account from the
>   input wallet is selected, only its addresses will be used as inputs.

Isn't it the same? :thinking:

Reply:
Okay, I read some other words in my head.